lib/models/recs.js

```diff
@@ -0,0 +1,154 @@
+const User = require('../models/User');
+const fetch = require('node-fetch');
+
+function calculateTopThree(dataArray1, dataArray2) {
```

Comment:
`dataArray1` and `dataArray2`? 
The naming is too vague.

Reply:
I thought I would name it unspecific so I can reuse the function. I changed it to different names specific to this project now